src/uct/sm/scopy/knem/knem_ep.c

```diff
@@ -12,6 +12,25 @@
 #include <uct/sm/base/sm_iface.h>
 #include <ucs/debug/log.h>
 
+
+const struct {
+    int     write; /* - `false`: READ from the remote region
```

Comment:
may be use uct_scopy_tx_op here?

Reply:
done, now using `uct_scopy_tx_op` directly